### PR TITLE
fixed errors in BM7 notebook

### PR DESCRIPTION
**Pull request by stvdwtt**
Fixes two errors in BM7 that were noticed by Damien.

Fixes swapped Dirichlet boundary conditions and missing factors of pi for parameters B1, B2, and C2. Also added a missing link to the upload form.

Go to, http://random-cat-741.surge.sh/benchmarks/benchmark7.ipynb/ to review

These changes can be [viewed live][live-site].

**Remember to tear down the [live site][live-site] when merging
or closing this pull request.**

[live-site]: http://random-cat-741.surge.sh

---
This change is [Reviewable](https://reviewable.io/reviews/usnistgov/pfhub/741)

### CODE REVIEW

**Comment by stvdwtt** [comment]
As usual, I can't make the notebook on my computer. At the meeting I'll have to talk to one of you to see what I'm doing wrong.

**Comment by tkphd** [comment]
@stvdwtt visit the surge.sh site -- no need to build locally anymore.

File: benchmarks/benchmark7.ipynb [hunk 1]
```diff
@@ -269,10 +269,10 @@
     "|-----------|-------|\n",
     "| $\\kappa$  | 0.0004|\n",
     "| $A_1$     | 0.0075|\n",
-    "| $B_1$     | 8.0  |\n",
+    "| $B_1$     | 8.0 $\\pi$  |\n",
```
Comment (tkphd) [inline benchmarks/benchmark7.ipynb]: `$8.0 \\pi$`, otherwise there's an odd-looking gap; or, `$8 \\pi$`

File: benchmarks/benchmark7.ipynb [hunk 2]
```diff
@@ -269,10 +269,10 @@
...
     "| $A_2$     | 0.03   |\n",
-    "| $B_2$     | 22.0  |\n",
-    "| $C_2$     | 0.0625|"
+    "| $B_2$     | 22.0 $\\pi$  |\n",
```
Comment (tkphd) [inline benchmarks/benchmark7.ipynb]: `$22.0 \\pi$`, otherwise there's an odd-looking gap; or, `$22 \\pi$`

File: benchmarks/benchmark7.ipynb [hunk 3]
```diff
@@ -269,10 +269,10 @@
...
+    "| $C_2$     | 0.0625 $\\pi$|"
```
Comment (tkphd) [inline benchmarks/benchmark7.ipynb]: `$0.0625 \\pi$`, otherwise there's an odd-looking gap; or, `$5 \\pi/80$`

**Comment by stvdwtt** [comment]
Great!

File: benchmarks/benchmark7.ipynb [hunk 4]
```diff
@@ -298,7 +298,7 @@
...
+    "For at least three simulations that have $L_2$ errors in the range $[5\\times10^{-3}, 1\\times10^{-4}]$, save the effective mesh size and $L_2$ error in a CSV or JSON file. [Upload this file to the PFHub website](https://pages.nist.gov/pfhub/simulations/upload_form/) as a 2D data set with the effective mesh size as the x-axis column and the $L_2$ error as the y-axis column. Calculate the effective element size as the square root of the area of the finest part of the mesh for nonuniform meshes. For irregular meshes with continous distributions of element sizes, approximate the effective mesh size as the average of the square root of the area of the smallest 5% of the elements.\n",
```
Comment (tkphd) [inline benchmarks/benchmark7.ipynb]: Not sure why we ask benchmarkers to filter their data, why we're requesting
"effective mesh size," or why we ask people to upload CSV files directly to
PFHub instead of an archive. Simplify:

    Save the effective element size $h$ and $L_2$ error for each simulation.
    [Archive this data](https://github.com/usnistgov/pfhub/issues/491) in a
    CSV or JSON file, using one column (or key) each for $h$ and $L_2$. 
    Calculate the effective element size as the square root of the area of
    the finest part of the mesh for nonuniform meshes. For irregular meshes
    with continuous distributions of element sizes, approximate the effective
    element size as the average of the square root of the area of the smallest
    5% of the elements. Then [submit your results on the PFHub website]
    (https://pages.nist.gov/pfhub/simulations/upload_form/).

Comment (stvdwtt) [reply benchmarks/benchmark7.ipynb]: @tkphd I fixed the "pi"s as requested. I modified the archive/upload section along the lines requested, and made similar modifications to related sections further on.

I kept sections describing what to list as the x axis and y axis while submitting the results, because I assume we want that standardized.

**Comment by tkphd** [comment]
Let's just be sure: we want 11 periods for `x ϵ [0,1)` and one-fourth of a period for `t ϵ [0,8]`, correct?

**Comment by wd15** [comment]
Just noticed that on the website,

https://pages.nist.gov/pfhub/benchmarks/benchmark7.ipynb/#Parameter-values

and on Surge, 

http://random-cat-741.surge.sh/benchmarks/benchmark7.ipynb/#Parameter-values

the mathematics in the table is not being rendered.

**Comment by tkphd** [comment]
Huh? The Surge table looks correct in my browser (Chrome).

**Comment by wd15** [comment]
Right, myy browser is doing weird things with all the benchmarks.

**Comment by stvdwtt** [comment]
@tkphd Correct.

**Comment by tkphd** [comment]
@wd15, Firefox does not render the math. The console reports
> "WARNING: cdn.mathjax.org has been retired. Check https://www.mathjax.org/cdn-shutting-down/ for migration tips."  MathJax.js:32:5

**Comment by wd15** [comment]
@tkphd: created issue for maths in firefox, #747
